src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/ContentPicker/ContentPickerDialog.jsx

```diff
@@ -31,6 +34,59 @@ const ContentPickerDialog = ({
     t,
     pickerConfig
 }) => {
+    const [site, setSite] = useState(editorContext.site);
+
+    const {data, error, loading} = useQuery(SiteNodesQuery, {
+        variables: {
+            query: 'select * from [jnt:virtualsite] where ischildnode(\'/sites\')',
```

Comment:
:o SQL in the front-end via a a graphql pipeline. I've seen everything in this world ^^

Reply:
hahaha